<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>368</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>6</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_semok_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="huncancel_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기(저장취소)</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hunsave_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>57</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>지출 항</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_costsemok_widget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_costsemok_tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string>기존 등록된 세목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="new_costsemok_tableWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>신규등록 세목</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 지출 세목  등록</string>
   </property>
  </widget>
  <widget class="QComboBox" name="costhang_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>371</x>
     <y>53</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>83</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>신규등록 지출 세목</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox_widget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>54</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>57</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>84</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>지출 목</string>
   </property>
  </widget>
  <widget class="QComboBox" name="costmok_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="semok_delete_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 행 삭제</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="semok_delete_button_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 행 수정</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>gubun_comboBox_widget</tabstop>
  <tabstop>costhang_comboBox_widget</tabstop>
  <tabstop>costmok_comboBox_widget</tabstop>
  <tabstop>new_costsemok_widget</tabstop>
  <tabstop>add_semok_button</tabstop>
  <tabstop>hunsave_button</tabstop>
  <tabstop>huncancel_button</tabstop>
  <tabstop>registed_costsemok_tableWidget</tabstop>
  <tabstop>new_costsemok_tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>add_semok_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costsemok_reg_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hunsave_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costsemok_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>huncancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costsemok_save_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>semok_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>semok_delete_button_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costsemok_modify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>costsemok_reg_input()</slot>
  <slot>costsemok_save()</slot>
  <slot>costsemok_save_cancel()</slot>
  <slot>registed_costsemok()</slot>
  <slot>delete_selected_row()</slot>
  <slot>costsemok_modify()</slot>
 </slots>
</ui>
